client/stylesheets/_toCat.scss

```diff
@@ -4575,7 +4583,7 @@ On click the images will be displayed at normal size to complete the effect
 
 .searchResultsCards {
     vertical-align: top !important;
-    margin-top: -1015px !important;
+    margin-top: -965px !important;
```

Comment:
!important should not be used<br>Line should be indented 2 spaces, but was indented 4 spaces